src/ServiceControl.Config/Features.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+
+namespace ServiceControl.Config
+{
+    using System.Collections.Concurrent;
+    using System.Collections.Specialized;
+    using System.Configuration;
+    using Autofac;
+
+    [AttributeUsage(AttributeTargets.Property)]
+    public class FeatureToggleAttribute : Attribute
+    {
+        public string Feature { get; }
+
+        public FeatureToggleAttribute(string feature)
+        {
+            Feature = feature;
+        }
+    }
+
+    public static class Feature
+    {
+        public const string MonitoringInstances = "MonitoringInstances";
+    }
+
+    public class FeatureToggles
+    {
+        private ConcurrentDictionary<string, bool> features = new ConcurrentDictionary<string, bool>(StringComparer.InvariantCultureIgnoreCase);
```

Comment:
Doesn't seem like the dictionary part nor the concurrent part is necessary. Maybe a HashSet?